docs-source/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Liveness and readiness probes customization"
+date: 2019-02-23T20:58:51-05:00
+draft: false
+weight: 6
+description: "This document describes how to customize the liveness and readiness probes for WebLogic Server instance Pods."
+---
+
+This document describes how to customize the liveness and readiness probes for WebLogic Server instance Pods.
+
+#### Contents
+
+* [Liveness probe customization](#liveness-probe-customization)
+* [Readiness probe customization](#readiness-probe-customization)
+
+#### Liveness probe customization
+
+The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container. 
+
+The liveness probe interval and timeout can be customized using the `livenessProbe` attribute under the `serverPod` element of the domain resource. 
+
+Following is an example configuration to change liveness probe interval and timeout value.
+```
+  serverPod:
+    livenessProbe:
+      periodSeconds: 30
+      timeoutSeconds: 10
+```
+
+It is also possible to customize the liveness probe by specifying a custom script which will be invoked by the existing liveness probe script (`livenessProbe.sh`) after the existing script performs its own checks. This custom script can be specified either by using the `livenessProbeCustomScript` attribute in the domain resource or by setting `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable using `env` attribute under the `serverPod` element (see configuration examples below). If the custom script fails with non-zero exit status, the liveness probe will fail and Kubernetes will restart the container.
```

Comment:
(suggested re-write of the first sentence) -> After the liveness probe script (`livenessProbe.sh`) performs its checks, you can to customize the liveness probe by specifying a custom script, which will be invoked by `livenessProbe.sh`.  
This custom script can be specified either by -> You can specify the custom script either by
or by setting `LIVENESS_PROBE_CUSTOM_SCRIPT` environment -> or by setting the `LIVENESS_PROBE_CUSTOM_SCRIPT` environment
using `env` attribute -> using the `env` attribute
(see configuration examples below) -> (see the configuration examples)
with non-zero exit -> with a non-zero exit